src/Sulu/Bundle/TrashBundle/Application/TrashItemHandler/SupportableTrashItemHandlerInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TrashBundle\Application\TrashItemHandler;
+
+interface SupportableTrashItemHandlerInterface
```

Comment:
Yes normally I would think the same, but since most bundles will implement both `StoreTrashItemHandlerInterface` and `RestoreTrashItemHandlerInterface` in a single class, i thought there should also be a single declaration for the `supports()` method. What about renaming the interface to `TrashItemHandlerInterface`?

Reply:
Don't see any usage of this Interface so I would really just move the method to both interfaces. This seperate interface has not advantage in my view and will not be used in our application logic.